src/test/java/com/thangle/domain/user/UserServiceTest.java

```diff
@@ -0,0 +1,88 @@
+package com.thangle.domain.user;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Optional;
+
+import static com.thangle.fakes.UserFakes.buildUser;
+import static com.thangle.fakes.UserFakes.buildUsers;
+import com.thangle.persistence.user.UserStore;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+@ExtendWith(MockitoExtension.class)
+
```

Comment:
Annotation is a part of thedeclarion (event class, method, field ...) so don't put empty line here